pkgs/development/libraries/librecast/default.nix

```diff
@@ -0,0 +1,32 @@
+{
+  stdenv,
+  pkgs,
+  fetchFromGitea,
+  lcrq,
+  lib,
+  libsodium,
+  ...
+}:
+stdenv.mkDerivation rec {
+  name = "librecast";
+  version = "0.7-RC3";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "librecast";
+    repo = "librecast";
+    rev = "v${version}";
+    sha256 = "sha256-AD3MpWg8Lp+VkizwYTuuS2YWM8e0xaMEavVIvwhSZRo=";
```

Comment:
Prefer `hash = ...` to `sha256 = ...` Both currently work, but the former will make it easier if the algorithm ever needs to change.

Since there could be different hashing algorithms in use, this make sense.